### ISSUE TITLE
TD & CO demo skirmish crash

### ISSUE THREAD

**Issue by JohnsterID** [TheAssemblyArmada/Vanilla-Conquer#723]
Testing TD demo and CO demo to play Skirmish.

1. Multiplayer Game
2. Skirmish
3. Click Ok with map Monkey In The Middle selected.
4. Crash

[vanilla-conquer-win-msvc-x86_64-edb8391.zip](https://github.com/TheAssemblyArmada/Vanilla-Conquer/releases/download/latest/vanilla-conquer-win-msvc-x86_64-edb8391.zip)
DEMO R725 EDB8391
DEMO.MIX
DEMOL.MIX
DEMOM.MIX
SC-002.MIX
SOUNDS.MIX
SPEECH.MIX

[crashdump.zip](https://github.com/TheAssemblyArmada/Vanilla-Conquer/files/7797360/crashdump.zip)

Unhandled exception at 0x000007FEF85D146C (vcruntime140.dll) in vanillatd.exe: 0xC0000005: Access violation reading location 0x0000000000000000.
vanillatd.exe!DisplayClass::Init_Theater(TheaterType theater) Line 406
0x00000000002fec78 "DESERT.PAL"
THEATER_DESERT (0 '\0')
No DESERT.PAL in the demo files causes the crash.

**Comment by OmniBlade** [comment]
Not sure what can be done about this other than document that the limitations of using demo data and the covertops data.

**Comment by JohnsterID** [comment]
Relevant error message (missing data file) instead of the application crashing?

**Comment by OmniBlade** [comment]
Fair point, the crash on missing data needs fixing for other issues as well.

**Comment by giulianobelinassi** [comment]
Isn't there a demo version with Nod files? it may be a requirement for getting CovOps demo to work.
